src/pages/components/button/usage.mdx

```diff
@@ -102,186 +132,419 @@ import { Add16 } from "@carbon/icons-react";
+  gutter between the two, without gutter hang.
+</Caption>
 
 </Column>
 </Row>
 
-## Labels
+<DoDontRow>
+  <DoDont caption="Do apply the same width to all buttons in a group, even if they don’t bleed.">
+
+![Do apply the same width to all buttons in a group](images/button_usage_21.png)
 
-Button labels should clearly indicate the action of the button. Use active verbs, such as **Add** or **Delete**. Use sentence-style capitalization (only the first word in a phrase and any proper nouns capitalized) and no more than three words for button labels.
+  </DoDont>
+  <DoDont type="dont" caption="Do not make buttons in a group different widths.">
 
-For sets of buttons, use specific labels, such as **Save** or **Discard**, instead of using **OK** and **Cancel**. This is particularly helpful when the user is confirming an action.
+![Do not make buttons in a group different widths.](images/button_usage_22.png)
 
-For consistency, see [labels and idioms](/guidelines/content/glossary) for the approved list of action labels.
+  </DoDont>
+</DoDontRow>
 
-## Icon usage
+#### Horizontally arranged groups
 
-- Use glyphs (16px) within buttons.
-- Glyphs are distinguished by their solid shape and knocked-out details.
-- Glyphs should always appear to the right of the text.
-- Glyphs used in buttons must be directly related to the action that the user is taking.
-- Glyphs must be the same color value as the text within a button.
-- Ghost buttons require a glyph icon.
+When using multiple buttons, the position of the primary button adheres to the [alignment guidance](#Alignment) above. To sum up, a primary button will be left-aligned and positioned to the left of the secondary/tertiary button on full-page designs. The primary button will be right-aligned and appear to the right of the secondary/tertiary button within wizards and dialog windows.
+
+<InlineNotification>
+
+**Note:** This guidance has changed. Previously we advocated maintaining the primary button position to the right of the secondary button when the button group was left-aligned. After talking to teams and doing more research, we’ve revised our position.
+
+</InlineNotification>
 
 <Row>
 <Column colLg={8}>
 
-![button with glyph](images/button-usage-1.png)
+![Button groups in context](images/button_usage_23.png)
 
-<Caption>Glyph usage in buttons</Caption>
+</Column>
+</Row>
+
+#### Stacked button groups
+
+Typical landing pages for product have buttons side by side. However vertical button groups are also common in products, to save real estate in narrow columns and occasionally side panels. In these instances, the primary button is always on top and the secondary or tertiary button is below.
+
+Generally, when designers stack buttons, they tend to use the fixed buttons. However, stacked fluid buttons are also an option in desktop side panels with especially long calls to action. 
+
+<Row>
+<Column colLg={12}>
+
+![Fluid and fixed stacked button group examples](images/button_usage_24.png)
```

Comment:
I think the first example (fixed) can totally be implemented today, I believe the fluid one would require overrides

Reply:
I added this copy:

Generally, when designers stack buttons, they tend to use the fixed buttons. However, stacked fluid buttons are also an option in desktop side panels with especially long calls to action. _Note: experimenting with stacked fluid buttons would require an override to the existing code._